### DISCUSSION TITLE
finn creating dataflow partition error

### DISCUSSION THREAD

**Discussion by mustafatasci42** [Xilinx/finn#684] [Q&A]
I trained the lenet brevitas model in the figure and exported it as onnx. 

![errorfinn](https://user-images.githubusercontent.com/15524481/190850261-08f486a4-e358-4a24-99be-8d3325649def.png)

I imported this onnx file to bnn_pynq / tfc end_to_end jupyter file in Finn notebooks. Everything worked fine until Creating a Dataflow Partition. At this stage, the code **_parent_model = model.transform(CreateDataflowPartition())_** gives an error. 
[Lenet_Mnist_w8_a8.zip](https://github.com/Xilinx/finn/files/9591370/Lenet_Mnist_w8_a8.zip)

**Comment by MFJI** [comment]
I also have an error when I use Finn0.8.1, but I don't have an error when I use Finn0.6.0.
